CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ internal API changes are not present.
 Main (unreleased)
 -----------------
 
+v0.36.0-rc.0 (2023-08-25)
```

Comment:
The [instructions](https://github.com/grafana/agent/blob/main/docs/developer/release/3-update-version-in-code.md) say:

> Delete the Main (unreleased) header and anything underneath it as part of the cherry-pick. Alternatively, do it after the cherry-pick is completed.

However, this isn't done in the [example PR](https://github.com/grafana/agent/pull/3066/files) from the instructions 😄  Maybe you could do it for the stable, non-RC version. I wouldn't bother with that for now.

Reply:
I'm refering to deleting the "Main (unreleased)" completely

Reply:
That delete is for next step further. 
What I'm doing now is this:
![image](https://github.com/grafana/agent/assets/17101802/24ecda47-f627-4e20-a357-051af82b297c)

Reply:
Note that your example PR is merging into release branch and not main.